Magento_GiftRegistry/templates/address/edit.phtml

```diff
@@ -1,11 +1,14 @@
 <?php
-
-// phpcs:disable Magento2.Templates.ThisInTemplate
-// phpcs:disable Magento2.Files.LineLength.MaxExceeded
-// phpcs:disable Squiz.ControlStructures.ControlSignature.NewlineAfterOpenBrace
-// phpcs:disable Squiz.WhiteSpace.ScopeClosingBrace.ContentBefore
-
+use Magento\Customer\Block\Widget\Name;
+?>
+<?php
```

Comment:
maybe we can keep it in one `<?php ?>` tag?

Reply:
<img src=https://www.probuzzing.com/wp-content/uploads/2015/11/Obama-Yes-We-Can.jpg />